data/map.txt

```diff
@@ -32289,6 +36502,17 @@ planet Yedikule
 	government Uninhabited
 	security 0
 
+planet "Yniu Eiu"
+	landscape land/canyon0
+	description `Yniu Eiu is a rocky world with many canyons. A large portion of the surface is bright shades of gray and white, but the more important visual are patches of purple-red cellular lakes flowing to and through other cells, canyons, and valleys. It is unclear if this is a natural occurrence or if these were artificially created, but the way the areas around these sights are dug out and shaped imply that this planet used to be a home to some type of civilization, or one that found importance in this material.`
```

Comment:
What do you mean by cells in this context?
```suggestion
	description `Yniu Eiu is a rocky world with many canyons. A large portion of the surface is bright shades of gray and white, but the more striking visuals are the patches of purple-red cellular lakes flowing to and through other cells, canyons, and valleys. It is unclear if this is a natural occurrence or if these were artificially created, but the way the areas around these sights are dug out and shaped imply that this planet used to be a home to some type of civilization, or one that found importance in this material.`
```

Reply:
Groups of other lakes, I suppose it's a bit too generic to use "cells"